mopidy_mpd/protocol/music_db.py

```diff
@@ -84,6 +87,65 @@ def _artist_as_track(artist):
     )
 
 
+def _get_art(context, uri=None, offset=0):
+    # TODO work out how validators work and move these there
+    if uri is None:
+        raise exceptions.MpdArgError("Need to specify uri")
+    offset = protocol.INT(offset)
+
+    images = context.core.library.get_images([uri]).get()[uri]
+
+    if len(images) == 0:
+        raise exceptions.MpdNoExistError("No file exists")
+
+    image_uri = images[0].uri
+
+    if image_uri == context.art_cache[0]:
+        bytes = context.art_cache[1]
```

Comment:
This variable assignment is shadowing a python builtin, which should be picked up by flake8-builtins as an error. Even if it isn't, it would be best to change it.